### ISSUE TITLE
Ip Bloqueada

### ISSUE THREAD

**Issue by ignaciop000** [martjanz/precios-claros#2]
Con la configuración inicial, no es posible realizar el scraping ya que te bloquea la ip y no es posible continuar descargando los datos

**Comment by martjanz** [comment]
Sí, según lo que probé bloquean después de una serie de requests. Quizás se puede probar con IPs rotativas o espaciando más las requests, no tirarlas todas juntas.
